applications/dashboard/models/class.usermodel.php

```diff
@@ -854,7 +864,7 @@ public function connect($uniqueID, $providerKey, $userData, $options = []) {
 
         if ($userID) {
             // Save the user.
-            $this->syncUser($userID, $userData);
+            $this->syncUser($userID, $userData, false /*force*/, $isTrustedProvider);
```

Comment:
Documenting the names of parameters as part of a method call? This looks like a new practice. Is it new? It looks new.